src/Inviqa/StockIndicatorExport/Domain/Model/StockIndicator.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Inviqa\StockIndicatorExport\Domain\Model;
+
+use InvalidArgumentException;
+
+final class StockIndicator
+{
+    private const LOW_STOCK_THRESHOLD = 10;
+
+    /** @var string */
+    private $value;
+
+    public static function fromString(string $value): StockIndicator
+    {
+        if (!in_array($value, ['RED', 'YELLOW', 'GREEN'])) {
+            throw new InvalidArgumentException(
+                'Stock indicator can only be RED, YELLOW or GREEN, but ' . $value . ' given'
+            );
+        }
+
+        return new self($value);
+    }
+
+    public static function red(): StockIndicator
+    {
+        return new self('RED');
+    }
+
+    public static function yellow()
+    {
+        return new self('YELLOW');
+    }
+
+    public static function green()
+    {
+        return new self('GREEN');
+    }
+
+    public static function for(Product $product): StockIndicator
```

Comment:
Maybe this should be `fromProduct()`?

Reply:
I was following the sentence in the scenario "export stock indicator _for_ the product"
but yep, maybe it is a bit weird :)